tests/lowering/pool/test_max_pool_2d.py

```diff
@@ -0,0 +1,43 @@
+import torch
+import torch_ttnn
+import pytest
+import ttnn
+
+from tests.utils import assert_with_pcc
+
+
+class MaxPool2dModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, *args, **kwargs):
+        return torch.nn.functional.max_pool2d(*args, **kwargs)
+
+
+@pytest.mark.parametrize(
+    "input_shapes, kernel_size, return_indices",
+    (
+        ((1, 3, 64, 64), (3, 3), False),
+        ((1, 3, 64, 64), (3, 3), True),
```

Comment:
I'm getting this error for both test cases.  Would you please provide a clue for debugging?
```
E                   torch._dynamo.exc.BackendCompilerFailed: backend='ttnn_backend' raised:
E                   Exception: An error occurred when running the 'ToTtPass' pass after the following passes: []
E                   
E                   
E                   You can suppress this exception and fall back to eager by setting:
E                       import torch._dynamo
E                       torch._dynamo.config.suppress_errors = True

../tt-metal/python_env/lib/python3.8/site-packages/torch/fx/passes/infra/pass_manager.py:296: BackendCompilerFailed
```

Reply:
What methods have you tried to use to debug this? Whenever I see this particular error in the pass_manager, I do something like:

```
pytest --trace # include other arguments if needed
```
Then enter the following:
```
(Pdb) b ../tt-metal/python_env/lib/python3.8/site-packages/torch/fx/passes/infra/pass_manager.py:296
(Pdb) r
# wait for the breakpoint to hit
(Pdb) interact
>>> import pdb
>>> pdb.post_mortem(e.__traceback__)
```
This should usually tell you the file and line number where the real error happened.

Reply:
Thanks!  I got this error message and guessed if `kwargs` was immutable.  I made `new_kwargs` and moved on to another error.
```
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> PDB runcall (IO-capturing turned off) >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
> /home/ubuntu/jdh8/pytorch2.0_ttnn/tests/lowering/pool/test_max_pool_2d.py(25)test_max_pool_2d()
-> m = MaxPool2dModule()
(Pdb) b ../tt-metal/python_env/lib/python3.8/site-packages/torch/fx/passes/infra/pass_manager.py:296
Breakpoint 1 at /home/ubuntu/jdh8/tt-metal/python_env/lib/python3.8/site-packages/torch/fx/passes/infra/pass_manager.py:296
(Pdb) r
> /home/ubuntu/jdh8/tt-metal/python_env/lib/python3.8/site-packages/torch/fx/passes/infra/pass_manager.py(296)__call__()
-> raise Exception(msg) from e
(Pdb) interact
*interactive*
>>> import pdb
>>> pdb.post_mortem(e.__traceback__)
> /home/ubuntu/jdh8/tt-metal/python_env/lib/python3.8/site-packages/torch/fx/immutable_collections.py(16)_no_mutation()
-> raise NotImplementedError(f"'{type(self).__name__}' object does not support mutation. {_help_mutation}")
```